This whole entity thing is a mess.

Add a temporary fix to shut up the compiler and leave appropriate TODO
comments.

The fact, that a prototype for a function in entity.c is in indep.h
speaks for themself.
---
 entity.c |    6 ++++++
 file.c   |    2 ++
 indep.c  |    5 +++++
 indep.h  |    1 -
 table.c  |    2 ++
 5 files changed, 15 insertions(+), 1 deletion(-)

--- a/entity.c
+++ b/entity.c
@@ -31,6 +31,12 @@ static char *alt_latin1[ 96 ] = {
 };
 /* *INDENT-ON* */
 
+/*
+ * TODO(rkta): Bogus protoyp.
+ * This should usually go to entity.h, but this header is auto-generated by
+ * the mktable programm.
+ */
+char * conv_entity(unsigned int c);
 char *
 conv_entity(unsigned int c)
 {
--- a/indep.c
+++ b/indep.c
@@ -13,6 +13,11 @@
 #include "myctype.h"
 #include "entity.h"
 
+/*
+ * TODO(rkta): See entity.c for an explanation why we need this protoype here.
+ */
+char * conv_entity(unsigned int c);
+
 unsigned char QUOTE_MAP[0x100] = {
     /* NUL SOH STX ETX EOT ENQ ACK BEL  BS  HT  LF  VT  FF  CR  SO  SI */
     24, 24, 24, 24, 24, 24, 24, 24, 24, 24, 24, 24, 24, 24, 24, 24,
--- a/indep.h
+++ b/indep.h
@@ -41,7 +41,6 @@ extern char *HTML_QUOTE_MAP[];
 #define html_quote_char(c) HTML_QUOTE_MAP[(int)is_html_quote(c)]
 
 extern size_t strtoclen(const char *s);
-extern char *conv_entity(unsigned int ch);
 extern int getescapechar(char **s);
 extern char *getescapecmd(char **s);
 extern char *allocStr(const char *s, int len);
--- a/file.c
+++ b/file.c
@@ -32,6 +32,8 @@
 
 Buffer * loadGeminiBuffer(URLFile *uf, Buffer *volatile newBuf);
 
+char * conv_entity(unsigned int c); /* TODO(rkta): Move to header */
+
 extern int fold_pre;
 static int frame_source = 0;
 static int need_number = 0;
--- a/table.c
+++ b/table.c
@@ -12,6 +12,8 @@
 #include "Str.h"
 #include "myctype.h"
 
+char * conv_entity(unsigned int c); /* TODO(rkta): Move to header */
+
 int symbol_width = 0;
 int symbol_width0 = 0;
 
